### PR TITLE
allow TimeSpan humanizations with numbers converted to words

### PR DESCRIPTION
**Pull request by stevedesmond-ca**
Since #405 appears abandoned, this covers the `TimeSpan` part of #290. I went with adding an optional `numbersToWords` parameter to it's `Humanize()` method as opposed to adding a `ToWords()` method, based on the consistency argument from the issue.

e.g. `TimeSpan.FromSeconds(2).Humanize(numbersToWords: true)` => `two seconds`

The `numbersToWords` param proliferated a little deeper than I would have liked, so I'm open to discussing other options if that's preferred.

### CODE REVIEW

**Comment by MehdiK** [comment]
This is a great addition. Thanks for the PR. You have a failing test. Could you please fix that?

**Comment by MehdiK** [comment]
I am a bit unsure about `numberToWords` argument name. What else could we call it? `toWords`, `inWords`, `words`?

**Comment by MehdiK** [comment]
@stevedesmond-ca could you please address the above comments? I will revisit this in two weeks and will close it off if not addressed by then. Thanks.

**Comment by stevedesmond-ca** [comment]
Sorry, not sure how this slipped my to-do list!

Parameter names have been updated to `toWords` and the "approved API" now includes the new optional parameters, so the API approver test passes.

File: src/Humanizer/Humanizer.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <Product>Humanizer ($(TargetFramework))</Product>    
     <UseFullSemVerForNuGet>true</UseFullSemVerForNuGet>
     <GenerateDocumentationFile Condition=" '$(Configuration)' == 'Release' ">true</GenerateDocumentationFile>
-    <SignAssembly>true</SignAssembly>
```
Comment (MehdiK) [inline src/Humanizer/Humanizer.csproj]: Why did you remove assembly signing?!

File: src/Humanizer.Tests/Humanizer.Tests.csproj [hunk 2]
```diff
@@ -24,7 +24,6 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="ApiApprover\PublicApiApprovalTest.approve_public_api.approved.txt" />
```
Comment (MehdiK) [inline src/Humanizer.Tests/Humanizer.Tests.csproj]: Could you please add the approve file back in?

**Comment by MehdiK** [comment]
Thanks for addressing the comments. You've changed a few things that shouldn't have been changed, as outlined in the new comments. Could you please address them too?

Also please explain the additional parameter in readme. Thanks.

**Comment by stevedesmond-ca** [comment]
Oops, yep, those `.csproj` changes definitely shouldn't have been committed!

Fixed, and `README` updated to include a blurb about the new `toWords` parameter.

**Comment by MehdiK** [comment]
Thanks. Merged.
